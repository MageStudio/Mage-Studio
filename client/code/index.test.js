import React from 'react';
import toJSON from 'enzyme-to-json';
import { shallow } from 'enzyme';

import Code from './index';

describe('Code', () => {

    it('should render fine', () => {
        const component = shallow(<Code />);
        expect(toJSON(component)).toMatchSnapshot();
    });
});
