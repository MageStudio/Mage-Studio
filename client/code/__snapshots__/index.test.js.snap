// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Code should render fine 1`] = `
<div
  className="app"
/>
`;
